Source/DafnyCore/Resolver/NameResolutionAndTypeInference/NameResolutionAndTypeInference.cs

```diff
@@ -5917,7 +5918,8 @@ public MethodCallInformation ResolveApplySuffix(ApplySuffix e, ResolutionContext
           }
           if (callee != null) {
             // produce a FunctionCallExpr instead of an ApplyExpr(MemberSelectExpr)
-            var rr = new FunctionCallExpr(e.Lhs.Tok, mse.MemberNameNode, mse.Obj, e.Tok, e.CloseParen, e.Bindings, atLabel) {
+            // TODO use e.Origin instead of e.Lhs.Origin
```

Comment:
What's at stake here between `e.Origin` and `e.Lhs.Origin`?

Reply:
When using `e.Lhs.Origin` the error is reported on the call target, so `foo` in `foo(a,b)`, but when using `e.Origin`, the error is reported on the `(`. This PR maintains the old behavior (which is inconsistent with call statements), and the TODO is meant for a follow-up.